privaterelay/urls.py

```diff
@@ -44,6 +44,7 @@
     ),
     path('accounts/', include('allauth.urls')),
     path('api/', include('api.urls')),
+    path('faq', views.faq, name='faq'),
```

Comment:
@Vinnl: We still need to define this url, because we have this line in our email-sending code:
```python
        attachment_msg = (
            'Firefox Relay supports email forwarding (including attachments) '
            'of email up to 150KB in size. To learn more visit {site}{faq}\n'
        ).format(site=settings.SITE_ORIGIN, faq=reverse('faq'))
```

It doesn't seem to cause any problem - the `/faq` url is still served with React UI. Just a heads-up.